pkg/util/namespace/namespace.go

```diff
@@ -3,23 +3,73 @@ package namespace
 // Copyright (c) Microsoft Corporation.
 // Licensed under the Apache License 2.0.
 
-import (
-	"strings"
-)
+// IsOpenShiftNamespace returns true if ns is a namespace in the defined hardcoded map
+func IsOpenShiftNamespace(ns string) bool {
+	GetNamespace := map[string]struct{}{
```

Comment:
> 2. `IsOpenShiftNamespace` currently creates a map on stack every time we call this function and it sounds like a waste for a hardcoded map of namespaces. Probably worth using `switch-case` here instead.

Thanks for your  suggestions. I think Golang map is just a hash table and it should be more efficient to find the particular string from time complexity's perspective. Although it can sacrifice some performance from space complexity's perspective, it's a really small hardcoded map and I think the cost can be ignored to be honest.